contracts/zero-ex/test/features/fund_recovery_tests.ts

```diff
@@ -0,0 +1,126 @@
+import {
+    blockchainTests,
+    constants,
+    describe,
+    expect,
+    getRandomPortion,
+    provider,
+    randomAddress,
+} from '@0x/contracts-test-utils';
+import { BigNumber, hexUtils } from '@0x/utils';
+import { artifacts as erc20Artifacts, DummyERC20TokenContract } from '@0x/contracts-erc20';
+import { LogWithDecodedArgs } from 'ethereum-types';
+import { IOwnableFeatureContract, IZeroExContract, LiquidityProviderFeatureContract } from '../../src/wrappers';
+import {TestWethContract} from '../wrappers';
+import { artifacts } from '../artifacts';
+import { abis } from '../utils/abis';
+import { fullMigrateAsync } from '../utils/migration';
+import { TestInitialMigrationContract } from '../generated-wrappers/test_initial_migration';
+import { FundRecoveryFeatureContract } from '../generated-wrappers/fund_recovery_feature';
+import { Web3Wrapper } from '@0x/web3-wrapper';
+import { Address } from '@0x/utils/lib/src/abi_encoder';
+
+    blockchainTests('FundRecovery', async env => {
+        
+        let owner: string;
+        let designatedAddress: string;
+        let testAddress: string;
+        let zeroEx: IZeroExContract;
+        let token: DummyERC20TokenContract;
+        let weth: TestWethContract;
+        let feature: FundRecoveryFeatureContract;
+
+        before(async () => {
+            env.blockchainLifecycle.startAsync();
+            const ZERO_EX_EXCHANGE_PROXY = '0xdef1c0ded9bec7f1a1670819833240f027b25eff';
+            [owner, designatedAddress, testAddress] = await env.getAccountAddressesAsync();
+            
+            console.log(owner);
+            //EP Migration
+            zeroEx = await fullMigrateAsync(owner, env.provider, env.txDefaults, {});
```

Comment:
This function will deploy the exchange proxy at a different address than `0xdef1c0ded9bec7f1a1670819833240f027b25eff`, so you should use `zeroEx.address` instead of `ZERO_EX_EXCHANGE_PROXY` in these tests.